helper/common_handler.js

```diff
@@ -3038,13 +3038,13 @@ async function ganttChartObject(data) {
                     }
 
                     dataArray.push(obj);
-                } else if (ele.status.toLowerCase() == "not yet started" || ele.status.toLowerCase() == "not started yet") {
+                } else if (ele.status.toLowerCase() == "not yet started" || ele.status.toLowerCase() == "not started yet" || ele.status.toLowerCase() == "notstarted") {
```

Comment:
@rakeshSgr - We don't need to handle previous status with space

Reply:
its existing pdf old unnati report code

Reply:
@rakeshSgr - But we don't need that right anymore